### PR TITLE
Update credential-guard-manage.md

### PR DESCRIPTION
**Pull request by botmoto**
Added Server 2019
Added registry key for disabling Credential Guard
Separated the process for disabling Credential Guard from virtualization-based security
Changed bcdedit entries to vsmlaunchtype instead of hypervisorlaunchtype, as this would avoid disabling a hypervisor if already in use.

### CODE REVIEW

File: windows/security/identity-protection/credential-guard/credential-guard-manage.md [hunk 1]
```diff
@@ -150,9 +151,13 @@ To disable Windows Defender Credential Guard, you can use the following set of p
...
     - HKEY\_LOCAL\_MACHINE\\System\\CurrentControlSet\\Control\\LSA\LsaCfgFlags
-    - HKEY\_LOCAL\_MACHINE\\Software\\Policies\\Microsoft\\Windows\\DeviceGuard\\EnableVirtualizationBasedSecurity
-    - HKEY\_LOCAL\_MACHINE\\Software\\Policies\\Microsoft\\Windows\\DeviceGuard\\RequirePlatformSecurityFeatures
-
+    - HKEY\_LOCAL\_MACHINE\\Software\\Policies\\Microsoft\\Windows\\DeviceGuard\\LsaCfgFlags
+> [!NOTE]
+> If you also wish to disable virtualization-based security delete the following registry settings:
+```syntax
+HKEY\_LOCAL\_MACHINE\\Software\\Policies\\Microsoft\\Windows\\DeviceGuard\\EnableVirtualizationBasedSecurity
+HKEY\_LOCAL\_MACHINE\\Software\\Policies\\Microsoft\\Windows\\DeviceGuard\\RequirePlatformSecurityFeatures
```
Comment (illfated) [inline windows/security/identity-protection/credential-guard/credential-guard-manage.md]: Maybe you are unaware of this, but it is important to keep the existing Code Style of the document, especially when adding text inside an already formatted page. The missing indents for the lines in this section does not translate well in the formatted view (https://github.com/MicrosoftDocs/windows-itpro-docs/pull/3198/files?short_path=6b827b7#diff-6b827b75f925b9e41f8f06758fc3a0df), making it look like changes were made in lines where there are no changes in code view.

Comment (botmoto) [reply windows/security/identity-protection/credential-guard/credential-guard-manage.md]: Thanks for letting me know I've made some adjustments

**Comment by mypil** [comment]
@DaniHalfin - Can you please assist to check, approve and merge the content for this PR? 

Thank you.

**Comment by illfated** [comment]
@botmoto : Good job with the changes. The PR content is a lot clearer to read now.

**Comment by illfated** [comment]
Oh. I just noticed something at the top of the file, the internal description is not marked as a comment field. There are 3 dashes (`---`) in line 1 and 16, but those dashes need to be replaced by 3x tilde (`~~~`) or 3x back tick **```** in order to work as comment block fences (at least here on Github), which I presume should result in the comment field not being present in the docs.microsoft.com page (or maybe be translated into some form of HTML comment field).

@mypil : Do you think this kind of change needs a separate PR, seeing that this is a separate issue from what this PR set out to resolve? Or could it be included in this PR?

**Comment by mypil** [comment]
@illfated - If it is for the same article then I think it can be done on the same PR. Since the PR content is not yet merged, you may be able to suggest the changes you want as a new commit so the PR creator can merge the new commit. Afterwards, then we can tag the doc owner to review and merge the content of the entire PR. What do you think?

**Comment by illfated** [comment]
Fair enough. My only concern is that I should check one or more of the other doc pages to see if the top comment section is delimited by Github MarkDown code to keep the current codestyle and not create a new issue during migration to the docs.microsoft.com page like I previously did when I changed an image name in one of my own PRs.

I will post back when I have checked this and suggest the change to the PR owner (**botmoto**).

**Comment by illfated** [comment]
Looks like the codestyle is kept the same for all 10 files in the directory
https://github.com/MicrosoftDocs/windows-itpro-docs/tree/master/windows/security/identity-protection/credential-guard
and it looks like the MetaData visibility is a random issue for 4 out of 10 pages (6 pages are OK on Github)
- [additional-mitigations.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/additional-mitigations.md)
- [credential-guard-considerations.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-considerations.md)
- [credential-guard-how-it-works.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-how-it-works.md)
- [credential-guard-known-issues.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-known-issues.md)
- [credential-guard-manage.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-manage.md)
- [credential-guard-not-protected-scenarios.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-not-protected-scenarios.md)
- [credential-guard-protection-limits.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-protection-limits.md)
- [credential-guard-requirements.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-requirements.md)
- [credential-guard-scripts.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard-scripts.md)
- [credential-guard.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/credential-guard/credential-guard.md)

Seeing that @DaniHalfin added a substantial amount of work in https://github.com/MicrosoftDocs/windows-itpro-docs/pull/2740
I prefer to leave it to him to decide what to do with this random issue.

**Comment by mypil** [comment]
@DaniHalfin - We would like to follow up. Can you please assist to check, approve and merge the content for this PR?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR? Thank you.

**Comment by AndreaBarr** [comment]
@Justinha Could you please approve and merge this PR?

**Comment by DaniHalfin** [comment]
@SteveSyfuhs - can you check please ?

**Comment by SteveSyfuhs** [comment]
Looks good to me,

**Comment by mypil** [comment]
@DaniHalfin - Kindly assist to merge the content of this PR. Thank you.

**Comment by mypil** [comment]
@AndreaBarr - Would you be able to merge the content of this PR? 

Thanks.

**Comment by illfated** [comment]
@mypil, @botmoto : The document is missing some formatting code outside of the affected lines so I can't comment directly in the code:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/3198/files#diff-6b827b75f925b9e41f8f06758fc3a0dfR132
This line belongs to the `> [!NOTE]` indicator 2 lines above and it should have looked like this to be correctly formatted:
```
> [!NOTE]
>
> For client machines that are running Windows 10 1703, LsaIso.exe is running whenever virtualization-based security is enabled for other features.
```
You can find the current public view of that text here:
- https://docs.microsoft.com/en-us/windows/security/identity-protection/credential-guard/credential-guard-manage#review-windows-defender-credential-guard-performance
(notice the single purple line with only **&#9432; Note** inside of it, followed by a blank white line and then the info paragraph in default text format.)

Would it be OK to include this cosmetic improvement in this PR, or do you prefer to have a separate PR for it?

**Comment by mypil** [comment]
@illfated -  I think it is better to include the cosmetic improvement in this PR but we would need @botmoto to add your suggested changes to this PR.

**Comment by botmoto** [comment]
@illfated @mypil Done. Also I used the two line format to be consistent with the rest of the document I don't think there is supposed to be 3 lines unless I'm mistaken.

**Comment by mypil** [comment]
@DaniHalfin @Justinha @mjcaparas @SteveSyfuhs @AndreaBarr - Would anyone of you be able to please merge the content of this PR?

Thank you.

**Comment by Justinha** [comment]
Thanks everyone. FYI ordered lists in markdown are automatically numbered correctly in HTML output, see https://daringfireball.net/projects/markdown/syntax#list

**Pull request merged**